geometry_msgs/Pose request
---
geometry_msgs/Pose nav_point
int64 recon_type
